lms/static/certificates/sass/_components.scss

```diff
@@ -537,7 +537,7 @@
 // #FOOTER
 // ------------------------------
 .footer-app {
-    border-top: rem(4) solid palette(grayscale, xx-trans);
+    border-top: rem(4) solid transparentize(palette(grayscale, base), 0.95);
```

Comment:
I'm not lightening the gray, I'm making it very transparent. I don't know why that makes sense for a border, though. I need to work out how to view this on a sandbox so I can see the effect.

BTW, I find using the name ``back`` for anything other than a background color to be confusing. Why didn't we go with ``light``? It isn't clear reading the code what ``back`` or ``x-back`` mean.